Header Vectornav
sensor_msgs/Imu imu
sensor_msgs/MagneticField mag_field